pike/model.py

```diff
@@ -1870,21 +1870,58 @@ def get_symlink(self, file):
         return self.connection.transceive(
                 self.get_symlink_request(file).parent.parent)[0]
 
-    def enumerate_snapshots_request(self, fh, max_output_response=16384):
-        smb_req = self.request(obj=fh.tree)
+    def create_fsctl_req(self, smb_req, fh, buflen=16384):
```

Comment:
recommend `fsctl_request` to match the naming convention in the rest of the file.

this helper doesn't have anything to do with the `create` operation, since it takes a file handle